### PR TITLE
Fix UTF8 handling

### PR DESCRIPTION
**Pull request by ignacioalles**
Fix issue #30 by storing un-trimmed value in last element reference.

### CODE REVIEW

**Comment by miqwit** [comment]
hello @ignacioalles. Any update on this?

**Comment by miqwit** [comment]
Ha ok. Now it's clear, thanks. I've made a code review above, mentioning some improvement if you don't mind to update your PR. I can do it myself too, if you're fine with that.

**Comment by ignacioalles** [comment]
I can't see your code review. Am I missing something? Also feel free to make any change you consider in my PR.

**Review by miqwit — CHANGES_REQUESTED** [review]

File: src/Controller/ErnParserController.php [hunk 1]
```diff
@@ -451,12 +452,13 @@ private function setCurrentElement($value) {
     // xml_parser is known to split values when encountering multibyte chars and call the character_data_handler multiple times
     if (!empty($this->lastElement) && $this->lastElement[0] === $elem && $this->lastElement[1] === $tag) {
       $value = $this->lastElement[2] . $value;
+      $value_clean = trim($value);
```
[the remark this answers is not in the document]
Comment (ignacioalles) [reply src/Controller/ErnParserController.php]: Sure!

**Comment by miqwit** [comment]
It's merged and released in 2.0.7 @ignacioalles. Thank you for your contribution.
I'd like to make progress on [another PR](https://github.com/miqwit/dedex/pull/29) but I'd need former version of DDEX ERN messages. If you happen to be able to provide some (anonymised, removing Party IDs and renaming artists and track names), let me know, it would be greatly appreciated.

**Comment by ignacioalles** [comment]
I don't have much information on samples of other ddex versions. I mainly use 3.8.2 but I think it is useful to have a way to support other versions without forking the entire project.

**Pull request merged**